shared/bls/spectest/fast_aggregate_verify_test.go

```diff
@@ -37,16 +37,25 @@ func testFastAggregateVerifyYaml(t *testing.T) {
 			test := &FastAggregateVerifyTest{}
 			require.NoError(t, yaml.Unmarshal(file, test))
 
-			pubkeys := make([]iface.PublicKey, len(test.Input.Pubkeys))
+			pubkeys := make([]common.PublicKey, len(test.Input.Pubkeys))
 			for j, raw := range test.Input.Pubkeys {
 				pkBytes, err := hex.DecodeString(raw[2:])
 				require.NoError(t, err)
 				pk, err := bls.PublicKeyFromBytes(pkBytes)
-				require.NoError(t, err)
+				if err != nil {
+					if test.Output == false && err == common.ErrInfinitePubKey {
+						return
+					}
+					t.Fatalf("cannot unmarshal pubkey: %v", err)
+				}
 				pubkeys[j] = pk
 			}
 
-			msgBytes, err := hex.DecodeString(test.Input.Message[2:])
+			msg := test.Input.Message
+			if msg == "" {
+				msg = test.Input.Messages
+			}
+			msgBytes, err := hex.DecodeString(msg[2:])
```

Comment:
Please add a comment or TODO why we have to do this. It should be fixed soon with https://github.com/ethereum/eth2.0-spec-tests/issues/22